firmware/Core/Src/telecommands/antenna_telecommand_defs.c

```diff
@@ -554,7 +554,7 @@ uint8_t TCMDEXEC_ant_report_antenna_deployment_activation_time(const char *args_
         return 4;
     }
 
-    uint16_t response;
+    uint32_t response;
```

Comment:
Please add units to variable name (i.e., `response_ms` or `activation_time_ms`).